runtime/oti/VMHelpers.hpp

```diff
@@ -2114,19 +2124,31 @@ class VM_VMHelpers
  *   another GC thread winning to scan(bit0/bit1,bit0 and bit1 is irrelevant and independent), again don't do anything, and let the winning thread do the work, instead
  */
 	static VMINLINE bool
-	tryWinningConcurrentGCScan(J9VMContinuation *continuation, bool isGlobalGC)
+	tryWinningConcurrentGCScan(J9VMContinuation *continuation, bool isGlobalGC, bool beingMounted)
 	{
-		uintptr_t complementGCConcurrentState = J9_GC_CONTINUATION_STATE_CONCURRENT_SCAN_NONE;
-		uintptr_t returnedState = J9_GC_CONTINUATION_STATE_CONCURRENT_SCAN_NONE;
+		volatile uintptr_t *localAddr = &continuation->state;
```

Comment:
we don'r really need this local - we have just 2 spots where we read it once and store it once via atomic, so we could just use continuation->state directly